packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -68,6 +163,46 @@ export const MicrophoneButton = (props: MicrophoneButtonProps): JSX.Element => {
       onRenderOffIcon={props.onRenderOffIcon ?? onRenderMicOffIcon}
       strings={strings}
       labelKey={props.labelKey ?? 'microphoneButtonLabel'}
+      menuProps={props.menuProps ?? generateDefaultDeviceMenuPropsTrampoline(props, strings)}
+      menuIconProps={!showDeviceSelectionMenuTrampoline(props) ? { hidden: true } : undefined}
+      split={showDeviceSelectionMenuTrampoline(props)}
     />
   );
 };
+
+/* @conditional-compile-remove-from(stable) meeting-composite control-bar-split-buttons */
+// Due to backward compatibility requirements, strings in MicrophoneButtonStrings must be
+// optional. So we fill in default values for missing ones.
```

Comment:
Instead of default values, can we make the params in generateDefaultDeviceMenuProps optional? It's confusing here that the likes of `cameraMenuTitle` will be populated, even with empty string, but (presumably) not shown

Reply:
Done.